testsupport/user_assertions.go

```diff
@@ -79,12 +74,9 @@ func VerifyResourcesProvisionedForSignup(t *testing.T, awaitilities wait.Awaitil
 		assert.NoError(t, err)
 
 		// Verify provisioned Identity
-		userID := userAccount.Spec.UserID
-		if !dns1123ValueRegexp.MatchString(userAccount.Spec.UserID) {
-			userID = fmt.Sprintf("b64:%s", base64.RawStdEncoding.EncodeToString([]byte(userAccount.Spec.UserID)))
-		}
+		identityName := identity.NewIdentityNamingStandard(userAccount.Spec.UserID, "rhd").IdentityName()
```

Comment:
I would expect the same logic as we use for creating the identities:
```go
userID := userAccount.Spec.UserID
if strings.ContainsAny(userAccount.Spec.UserID, ":/") {
	userID = fmt.Sprintf("b64:%s", base64.RawStdEncoding.EncodeToString([]byte(userAccount.Spec.UserID)))
}
```

Reply:
Ah, you reuse the logic from toolchain-common - didn't notice that, sorry :man_facepalming: